### PR TITLE
Added Set timeout, required to increase the timeout value.

### PR DESCRIPTION
**Pull request by onyxg**

### CODE REVIEW

**Comment by mimmi20** [comment]
@onyxg Please truncate the empty lines.

**Comment by onyxg** [comment]
@mimmi20 done.

**Comment by mimmi20** [comment]
@asgrim Could you check the test result on travis-ci?

**Comment by asgrim** [comment]
Hmm, not sure how to trigger the build here; the integration with Travis seemed to disappear; I've added it now, but I guess it would need to be pushed to trigger the build again :s

**Comment by asgrim** [comment]
The last build passed just fine, only whitespace changed since so should be fine.

File: src/BrowscapUpdater.php [hunk 1]
```diff
@@ -154,6 +154,20 @@ public function getLogger()
     }
 
     /**
+     * Sets the Connection Timeout
+     *
+     * @param $connectTimeout
```
Comment (asgrim) [inline src/BrowscapUpdater.php]: Type is missing from docblock, I believe it should be `@param int $connectTimeout`

**Pull request closed without merging**